src/main/java/cn/nukkit/item/Item.java

```diff
@@ -539,6 +546,23 @@ public CompoundTag getCustomBlockData() {
         return null;
     }
 
+    public Item setCustomBlockData(CompoundTag compoundTag) {
+        CompoundTag tags = compoundTag.copy();
+        tags.setName("BlockEntityTag");
+
+        CompoundTag tag;
+        if (!this.hasCompoundTag()) {
+            tag = new CompoundTag();
+        } else {
+            tag = this.getNamedTag();
+        }
+
+        tag.putCompound("BlockEntityTag", tags);
+        this.setNamedTag(tag);
+
+        return this;
+    }
+
```

Comment:
what exactly way the point of moving this method?

Reply:
Auto Code Cleanup on my IDE